src/runtime/bootstrap-custom-element.ts

```diff
@@ -67,16 +67,16 @@ export const proxyCustomElement = (Cstr: any, compactMeta: d.ComponentRuntimeMet
 
   const originalConnectedCallback = Cstr.prototype.connectedCallback;
   const originalDisconnectedCallback = Cstr.prototype.disconnectedCallback;
-  let hasHostListenerAttached = false;
   Object.assign(Cstr.prototype, {
+    hasHostListenerAttached: false,
```

Comment:
I wonder if it makes sense to prefix it with 2 underscore to highlight that it is an internal flag like `__registerHost`

```suggestion
    __hasHostListenerAttached: false,
```

Wdyt?

Reply:
definitely a good idea, thx for the suggestion